src/components/templates/address-form.tsx

```diff
@@ -18,10 +18,16 @@ export type AddressPayload = {
   phone: string | null
 }
 
+export declare enum Type {
+  SHIPPING = "shipping",
+  BILLING = "billing",
+  ADDRESS = "address",
```

Comment:
question: This one seams not being used, should we keep it?

Reply:
Wdym? I just added it and its being used in 4 files 😄

Reply:
I didn't mean the enum, but the address value 😂